src/main/java/com/continuum/nova/utils/Utils.java

```diff
@@ -7,699 +7,369 @@
-        locations.add(new ResourceLocation("blocks/tnt_bottom"));
-        locations.add(new ResourceLocation("blocks/tnt_side"));
-        locations.add(new ResourceLocation("blocks/tnt_top"));
-        locations.add(new ResourceLocation("blocks/torch_on"));
-        locations.add(new ResourceLocation("blocks/torch_on.mcmeta"));
-        locations.add(new ResourceLocation("blocks/trapdoor"));
-        locations.add(new ResourceLocation("blocks/trip_wire"));
-        locations.add(new ResourceLocation("blocks/trip_wire_source"));
-        locations.add(new ResourceLocation("blocks/vine"));
-        locations.add(new ResourceLocation("blocks/waterlily"));
-        locations.add(new ResourceLocation("blocks/water_flow"));
-        locations.add(new ResourceLocation("blocks/water_flow.mcmeta"));
-        locations.add(new ResourceLocation("blocks/water_still"));
-        locations.add(new ResourceLocation("blocks/water_still.mcmeta"));
-        locations.add(new ResourceLocation("blocks/web"));
-        locations.add(new ResourceLocation("blocks/wheat_stage_0"));
-        locations.add(new ResourceLocation("blocks/wheat_stage_1"));
-        locations.add(new ResourceLocation("blocks/wheat_stage_2"));
-        locations.add(new ResourceLocation("blocks/wheat_stage_3"));
-        locations.add(new ResourceLocation("blocks/wheat_stage_4"));
-        locations.add(new ResourceLocation("blocks/wheat_stage_5"));
-        locations.add(new ResourceLocation("blocks/wheat_stage_6"));
-        locations.add(new ResourceLocation("blocks/wheat_stage_7"));
-        locations.add(new ResourceLocation("blocks/wool_colored_black"));
-        locations.add(new ResourceLocation("blocks/wool_colored_blue"));
-        locations.add(new ResourceLocation("blocks/wool_colored_brown"));
-        locations.add(new ResourceLocation("blocks/wool_colored_cyan"));
-        locations.add(new ResourceLocation("blocks/wool_colored_gray"));
-        locations.add(new ResourceLocation("blocks/wool_colored_green"));
-        locations.add(new ResourceLocation("blocks/wool_colored_light_blue"));
-        locations.add(new ResourceLocation("blocks/wool_colored_lime"));
-        locations.add(new ResourceLocation("blocks/wool_colored_magenta"));
-        locations.add(new ResourceLocation("blocks/wool_colored_orange"));
-        locations.add(new ResourceLocation("blocks/wool_colored_pink"));
-        locations.add(new ResourceLocation("blocks/wool_colored_purple"));
-        locations.add(new ResourceLocation("blocks/wool_colored_red"));
-        locations.add(new ResourceLocation("blocks/wool_colored_silver"));
-        locations.add(new ResourceLocation("blocks/wool_colored_white"));
-        locations.add(new ResourceLocation("blocks/wool_colored_yellow"));
-        locations.add(new ResourceLocation("blocks/wwool_colored_pink"));
+        addResources(locations,
+                "blocks/anvil_base", "blocks/anvil_top_damaged_0",
```

Comment:
I'm tempted to reject this cause you've changed so much spacing for no apparent reason but if you put each texture name on a different line so it's easier to read then I'll accept this bit